Adventure/Assets/Scripts/RayCast.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RayCast : MonoBehaviour
+{
+    [Tooltip("Starting point f raycast used to detect interatives.")]
```

Comment:
Spelling mistake - "interactives"

Reply:
After making the changes, you're all good for merging.

Reply:
fixed the spelling mistake